components/PageControl/src/MDCPageControl.m

```diff
@@ -483,6 +483,11 @@ - (void)resetControl {
   _animatedIndicator = [[MDCPageControlIndicator alloc] initWithCenter:center radius:radius];
   [_containerView.layer addSublayer:_animatedIndicator];
 
+  // Move indicator to new position.
+  CGPoint point = [_indicatorPositions[_currentPage] CGPointValue];
+  [_animatedIndicator updateIndicatorTransformX:point.x - kPageControlIndicatorRadius];
```

Comment:
We are creating the _animatedIndicator just above.
Can we move this point calculation above and determine a proper center for:
```
_animatedIndicator = [[MDCPageControlIndicator alloc] initWithCenter:center radius:radius];
```
We may still need [_trackLayer resetAtPoint:].

Reply:
Done. `[_trackLayer resetAtPoint:]` didn't appear to be needed.